tests/robot_crawler_test.php

```diff
@@ -136,6 +136,14 @@ public function test_should_be_authenticated($expected, $myurl, $testurl) {
         $this->assertEquals((bool)$expected, $this->robot->should_be_authenticated($testurl));
         $this->resetAfterTest(true);
     }
+
+    /**
+     * Tests existence of new plugin parameter 'retentionperiod'
+     */
+    public function test_param_retention_exists() {
+        $param = get_config('local_linkchecker_robot', 'retentionperiod');
```

Comment:
I'll push the next revision without this test yet, but with other fixes. And I'll work on this unit test tomorrow.